javaparser-core/src/main/java/com/github/javaparser/ast/expr/Name.java

```diff
@@ -105,13 +115,22 @@ public Name setIdentifier(final String identifier) {
         }
         notifyPropertyChange(ObservableProperty.IDENTIFIER, this.identifier, identifier);
         this.identifier = identifier;
+        if (asStringOfName != null) {
+            asStringOfName.remove(this); // clear cached value for asString
+        }
```

Comment:
I'm afraid this isn't possible since it is part of generated code. The next run of the generators will wipe it out.

Reply:
I didn't know this was generated code. From reading the code I couldn't tell. Typically in those cases I see a special comment or so to warn not to modify generated code. 

Where do I learn more about the generator and how you build your code? Rather than subclassing and overriding extending the generator may be a different solution to make sure the enhancements keep included in the code.